compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -12882,6 +12882,88 @@ TR::Node *ificmpneSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier
    return node;
    }
 
+/**
+ * \brief
+ * Simplify add/subtract operations under integer comparison nodes
+ * to save an add/sub instruction during codegen.
+ *
+ * \details
+ *
+ * e.g. This following is a before-after comparison.
+ *
+ * Before:
+ * ificmple --> block_17 BBStart at n13n ()
```

Comment:
This example looks to have too much detail like the node numbers and block numbers - I thik abstracting those details out to make it more clear what the reader should be focusing on makes more sense.